src/Linux/local_cache.cpp

```diff
@@ -171,6 +175,7 @@ static void make_dir(const std::string& dirname, mode_t mode)
     {
         if (S_ISDIR(buf.st_mode))
         {
+            chmod(dirname.c_str(), mode);

```

Comment:
Please make sure to test that this works and does not error out when it is QGSD executing.